### PR TITLE
fix(domain): fixed status display

### PR DESCRIPTION
**Pull request by FredericEspiau**
### Description of the Change

Before : status switching displayed switching data and previous data
After : display only the switching data text

### CODE REVIEW

File: src/emailpro/domain/emailpro-domain.html [hunk 1]
```diff
@@ -67,26 +67,24 @@
...
-                                      data-ng-class="{'doing-grey': domain.taskInProgress}"></span>
+                                      data-ng-bind="tr('exchange_tab_domain_modify_in_progress', tr('exchange_tab_domain_inverse_NON_AUTHORITATIVE'))"></span>
                                 <span data-ng-if="domain.taskInProgress && !domain.partial && domain.state === stateOk && domain.type === domainTypeNonAuthoritative"
-                                      data-ng-bind="tr('exchange_tab_domain_modify_in_progress', tr('exchange_tab_domain_inverse_AUTHORITATIVE'))"
-                                      data-ng-class="{'doing-grey': domain.taskInProgress}"></span>
-                                <span data-ng-if="!domain.partial && domain.state === 'DELETING'"
+                                      data-ng-bind="tr('exchange_tab_domain_modify_in_progress', tr('exchange_tab_domain_inverse_AUTHORITATIVE'))"></span>
+                                <span data-ng-if="!domain.partial && (domain.state === 'DELETING' || domain.state === 'CREATING')"
```
Comment (cbourgois) [inline src/emailpro/domain/emailpro-domain.html]: Should use `stateCreating` and `stateDeleting` (availables in `$scope`)
Should remove test on `domain.partial`  (already tested in parent `td`)

Comment (FredericEspiau) [reply src/emailpro/domain/emailpro-domain.html]: I intend to create a service that will handle the states, just like I did for Exchange. For now, I focused on implementing the fix asap

Will do for the `domain.partial`, nice catch

**Comment by jleveugle** [comment]
@FredericEspiau some news from QC?

**Comment by FredericEspiau** [comment]
@jleveugle found the matching JIRA, it's ok

**Pull request merged**